tests/test_full_reqres_api_MiK.py

```diff
@@ -0,0 +1,168 @@
+import unittest
+import requests
+import pytest
+from random import randint
+from faker import Faker as Fake
+from Utils.colored_messages import yellow_on_red_back as bug_report
+from Utils.extract_from_json import json_extractor as json_ex
+
+'''Global variables'''
+BASE_URL = 'https://reqres.in/api/'
+ALL_USERS = 'users'
+REQUEST_PAGE = '?page='
+REQUEST_DELAYED = '?delay='
+RESOURCES = 'unknown'
+REGISTRATION = 'register'
+LOGIN = 'login'
+
+HTTP_STATUS_OK = 200
+HTTP_STATUS_CREATE = 201
+HTTP_STATUS_DELETE = 204
+HTTP_STATUS_UNSUCCESSFUL = 400
+HTTP_STATUS_NOT_FOUND = 404
+
+TEST_DATA = {
+    'name': Fake().name(),
+    'job':  Fake().job(),
+    "email": "eve.holt@reqres.in",
+    "password": Fake().password()
+}
+
+TEST_DATA_UPD = {
+    'name': Fake().name(),
+    'job': Fake().job()
+}
+
+ERR_MESSAGE = {
+    "error": "Missing password"
+}
+
+
+class TestReqresAPI(unittest.TestCase):
+
+    @pytest.mark.reqres
+    def test_1_check_list_users_pages(self):
+        """Getting the default list of users. Determining the number of pages and users."""
+        response = requests.get(f'{BASE_URL}{ALL_USERS}')
+        self.assertEqual(response.status_code, HTTP_STATUS_OK, bug_report('Bad request total user page.'))
```

Comment:
Now I got it! If test fail we will get a colored error message in terminal. Nice!